analyzers/dataframe/FCCAnalyses/Algorithms.h

```diff
@@ -6,6 +6,9 @@
 
 #include "edm4hep/ReconstructedParticleData.h"
 
+#include "FCCAnalyses/JetClusteringUtils.h"
+#include "FCCAnalyses/ReconstructedParticle.h"
+
```

Comment:
Can you move those includes to the .cc file?

Reply:
OK, I've updated the implementation. I still need JetClustering.h, though.